Source/Core/VideoCommon/TextureConversionShader.cpp

```diff
@@ -62,7 +62,7 @@ static void WriteHeader(ShaderCode& code, APIType api_type)
              "  float y_scale;\n"
              "  float gamma_rcp;\n"
              "  float2 clamp_tb;\n"
-             "  float3 filter_coefficients;\n"
+             "  uint3 filter_coefficients;\n"
```

Comment:
Does the switch to integers give us some advantage?  I only ask because I'd assume there'd be a (slight) performance hit by dividing by 64.0 in the shaders at each pixel.

Reply:
It allows using `combined_rows.rgb >> 6` to divide by 64 and round down (which is the hardware's behavior), and it also makes writing `texcol_raw &= 0x1ffu` easier.

I vaguely remember some annoyance about [`round` having implementation-defined behavior with .5](https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/round.xhtml), but looking at it further it's `floor` that would be needed here instead and that [doesn't have the same issue](https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/floor.xhtml) so I'm not sure if there's a reason why it *couldn't* be done with floats; it'd just be a bit messier. (Perhaps it was the part with YUV conversion, since that does have rounding?)